symphony/app/fbcnms-projects/magmalte/app/theme/design-system/Text.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
is this copied from the main `design-system` stuff? is that really a good idea? why don't we share?